languagetool-language-modules/ga/src/main/java/org/languagetool/rules/ga/DhaNoBeirtRule.java

```diff
@@ -65,9 +65,9 @@ public String getDescription() {
     String replacement = null;
     String msg = null;
     for (int i = 1; i < tokens.length; i++) {  // ignoring token 0, i.e., SENT_START
-      if (isNumber(tokens[i]) && (i < tokens.length - 2 && isPerson(tokens[i + 1]))) {
+      if (isNumber(tokens[i]) && (i < tokens.length - 1 && isPerson(tokens[i + 1]))) {
         markEnd = i + 1;
-        if ("dhá".equalsIgnoreCase(tokens[i].getToken()) && (i < tokens.length - 2)) {
+        if ("dhá".equalsIgnoreCase(tokens[i].getToken())) {
```

Comment:
Supposidly redundant. This check here may be the most important one, but I still got IndexOutOfBounds without it.

Reply:
I changed the earlier test so it should have equivalent behaviour